src/js/popup.js

```diff
@@ -503,6 +503,26 @@ function syncUISelections() {
 }
 
 /**
+* Get's the value of specific key within a url.
+* Temporary function for android while it doesn't support `browser_action.default_popup`
+*
+* Source: https://stackoverflow.com/a/901144/2662749
+* Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1330159
+*
+* @param  {String} key Key to get from url
+* @param  {String} url
+* @return {String}     Value of the key
+*/
+function getParameterByName(key, url) {
+  if (!url) { url = window.location.href; }
```

Comment:
I think we can use the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL) to do this nicely:

```javascript
getTabFromUrl(url) {
  return parseInt(new URL(url).searchParams.get('tabId'));
}
```

The nice thing about working on web extensions is that we only run on newer browsers, so we can use newer API's.